README.md

```diff
@@ -66,6 +67,8 @@ final componentList = page.components;
 
 final component = page.component('COMPONENT_ID');
 
+final incidentList = await statusPage.incidents;
+
```

Comment:
If useful, I could also add another example of use with the optional parameter: `final unresolvedIncidentList = statusPage.incidents('PAGE_ID', IncidentType.unresolved);`